config/environments/production.rb

```diff
@@ -68,7 +68,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = true
+  config.i18n.fallbacks = [I18n.default_locale]
```

Comment:
This is a new one on me 🤷‍♂

Reply:
setting it to `true` sets it to default_locale?

Reply:
Hi @mattrayner, thanks for the feedback. I was prompted by the command line to make this change but I can't remember exactly why it told me to do that, but it seemed like the right thing to do. Interestingly, I've just found this comment in applications.rb:

```
# The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
```

Reply:
Fair enough, maybe something has changed in the latest Ruby - what's the local version of Ruby you're running?

I ask because it might be worth updating the Docker image to use the same version

Reply:
I'm using version 2.6.1p33

```
$ ruby --version
// => ruby 2.6.1p33 (2019-01-30 revision 66950) [x86_64-darwin17]
```